include/networkit/linkprediction/LinkThresholder.hpp

```diff
@@ -44,7 +44,8 @@ std::vector<std::pair<node, node>> byCount(std::vector<LinkPredictor::prediction
  * @param percentageLinks Percentage of highest scored node-pairs to return
  * @return the first @a percentageLinks percent of the highest scores node-pairs
  */
-std::vector<std::pair<node, node>> byPercentage(std::vector<LinkPredictor::prediction> predictions, double percentageLinks);
```

Comment:
About `byCount`, I am not sure how can we avoid a copy here. Now we create a local copy of `predictions` so we can sort it.

Reply:
Yeah, you are right - for `byCount` the copy can not be avoided. My comment could have been clearer. The modification to the code was about shifting the copy inside the function, so that the interface can be unified.